dev/wlp-jakartaee-transform/rules/jakarta-bundles.properties

```diff
@@ -7,6 +7,7 @@ com.ibm.ws.security.javaeesec.cdi=io.openliberty.security.jakartasec.2.0.interna
 com.ibm.websphere.appserver.spi.jaspic=io.openliberty.jaspic.2.0.spi,,"Jakarta Authentication Service Provider Interface for Containers 2.0 SPI ","Jakarta Authentication Service Provider Interface for Containers 2.0 SPI"
 com.ibm.ws.security.jaas.common=io.openliberty.security.jaas.internal.common,,"Security JAAS Implementation","Security JAAS Implementation; Jakarta Enabled"
 com.ibm.ws.security.authentication.builtin=io.openliberty.security.authentication.internal.builtin,,"Security Built-in Authentication Implementation","Security Built-in Authentication Implementation; Jakarta Enabled"
+com.ibm.ws.ejbcontainer.security=io.openliberty.ejbcontainer.security.internal,,"EJB container Security Collaborator internal","EJB container Security Collaborator internal; Jakarta Enabled"
```

Comment:
Please check with Tracy to see if he thinks this should be called `io.openliberty.ejbcontainer.security.internal` or `io.openliberty.ejbcontainer.v40.internal.security`

Reply:
@tkburroughs - Any preferences in regards to Tom's comment above?

Reply:
@jvanhill No preference; this is not a project supported by my team.  The com.ibm.ws.ejbcontainer.security* projects are supported by the security team, I'd check with them.  From a strictly jakartaee perspective, that seems to be the direction we are headed.  Not sure exactly who on security teams supports these projects, but it appears the last person to modify them was @toshiyamamoto.

Reply:
Since this bundle is not / will not be specific to EJB 4.0, I would rather we leave the version out of it. I will leave it with this bundle name unless someone else has a strong reason to change it.

Reply:
@jvanhill Ah, after looking at this more, I agree.  There are 2 problems with adding in the `v40`;
1) the bundle really has nothing to do with EJB 4.0, as it would likely apply to all later versions as well; this change is strictly for jakarta rename, not ejb version.
2) If it did only apply to EJB 4.0, then it should be `io.openliberty.ejbcontainer.security.v40.internal`, as this belongs to the functional component `ejbcontainer.security`, not `ejbcontainer`.  Adding v40 early, and security at the end makes the bundle name very misleading.

Reply:
OK, thank you for checking.

Reply:
Oh, sorry to bring this up so late, but "EJB" is trademarked, so should probably be changed to "Jakarta Enterprise Bean container".